README.md

```diff
@@ -1,4 +1,7 @@
-# MQTT Blackbox Exporter
+# MQTT Blackbox Exporter [![Build Status](https://travis-ci.org/inovex/mqtt_blackbox_exporter.png?branch=master)](https://travis-ci.org/inovex/mqtt_blackbox_exporter)
+
+[![Go Report Card](https://goreportcard.com/badge/github.com/inovex/mqtt_blackbox_exporter)](https://goreportcard.com/badge/github.com/inovex/mqtt_blackbox_exporter)
```

Comment:
please change the target URL to https://goreportcard.com/report/github.com/inovex/mqtt_blackbox_exporter